packages/compiler-cli/src/ngtsc/docs/src/extractor.ts

```diff
@@ -23,6 +23,9 @@ import {extractTypeAlias} from './type_alias_extractor';
 
 type DeclarationWithExportName = readonly[string, ts.Declaration];
 
+const allowedPrivates = new Map([
+  [`ɵ$localize`, '$localize'],  // exported as private but available as global
+]);
```

Comment:
nit: I'd propose renaming this Map to indicate that we are not trying to expose private symbols, it's just a mechanism by which we exported some symbols as globals:
```suggestion
const exportedAsGlobals = new Map([
  [`ɵ$localize`, '$localize'],  // exported as private but available as global
]);
```